tainiisanta.js

```diff
@@ -0,0 +1,105 @@
+// есть n человек, необходимо распределить: кто кому дарит подарки
+// Нельзя дарить подарок самому себе
+
+function getRandomInt(max) { // при max = 3, результат одно из: 0,1,2
+    return Math.floor(Math.random() * max); // округляем число вниз и возвращаем полученное случайное число
+}
+
+function getLength (massiv) { // в этой программе из массивов удалять элементы я не буду, вместо этого буду заменять их значения на 0, а эта функция будет возвращать длину массива без "нулей".
+    let a=0;
+    for (let i = 0; i < massiv.length; i++) {
+        if (massiv[i] != 0) {
+            ++a;
+        }
+    }
+    return a;
+}
+
+function checkPairsEqual (i, rNumber) { // сравнивает два элемента массива, если они равно, то возвращает false (ты не можешь дарить подарки сам себе)
```

Comment:
you can compare 2 equal by ===. it witll return boolean value. This function is useless